src/utils/analytics/index.js

```diff
@@ -54,10 +54,12 @@ export const measureTime = (action = TIMED_ACTIONS.UNKNOWN, func, label) => asyn
  * @param label Optional label to describe the action.
  */
 export const recordTime = (action = TIMED_ACTIONS.UNKNOWN, t0, t1, label) => {
-    ReactGa.timing({
-        ...action,
-        value: t1 - t0,
-        label,
+    ReactGA.send({
+        hitType: "timing",
+        timingCategory: action.category,
+        timingVar: action.variable,
+        timingLabel: label,
+        timingValue: t1 - t0,
```

Comment:
I recall `react-ga4` having a specialized timing function, but maybe I'm mixing things up.

Reply:
I don't think so. There isn't any special timing feature in ga4 either, it's just a regular event